examples/lockdown/.thoth.yaml

```diff
@@ -1,15 +1,14 @@
 #
 # This is default Thoth configuration file placed in a root of a repo. Please adjust values listed below.
 
-host: test.thoth-station.ninja
+host: stage.thoth-station.ninja
```

Comment:
khemenu?

Reply:
stage make a little more obvious that this is a preview?!